test/t_cose_sign_verify_test.c

```diff
@@ -129,6 +132,14 @@ int_fast32_t sign_verify_basic_test()
     }
 #endif
 
+#if !defined(T_COSE_DISABLE_EDDSA) && defined(T_COSE_CRYPTO_LIB_OPENSSL)
+    /** Only OpenSSL supports EDDSA. */
+    return_value  = sign_verify_basic_test_alg(T_COSE_ALGORITHM_EDDSA);
+    if(return_value) {
```

Comment:
I'd prefer the test run even if its PSA crypto or EDDSA is disabled. The expected results should be the unknown algorithm error.

Someday there will be measured code coverage (I hope) and these tests are needed to have the complete coverage.

Reply:
Your comment has made me realize that `T_COSE_CRYPTO_LIB_OPENSSL` is not actually a preprocessor define but an enum value, which meant this test was never running.

There was actually no way to find out whether EDDSA is supported, so I picked up some of your changes from the dev branch to add the `t_cose_is_algorithm_supported`, and changed all the tests to use that. Let me know what you think of it.

I've added a separate test that checks for `ERR_UNSUPPORTED_SIGNING_ALG` instead.